ocaml/xapi/cli_util.ml

```diff
@@ -77,6 +77,8 @@ let result_from_task rpc session_id remote_task =
       | [] -> Failure (Printf.sprintf "Task failed but no error recorded: %s" (Ref.string_of remote_task)) in
     Backtrace.(add exn (t_of_sexp (Sexplib.Sexp.of_string trace)));
     raise exn
+  | `unknown ->
```

Comment:
It would help to add parantheses (or a begin/end pair) to make it obvious that this ``` `unknown``` is part of the `match Client.Task.get_status rpc session_id remote_task_with` rather than part of the `match error_info with`.